src/main/java/com/cozyhome/onlineshop/reviewservice/service/ReviewServiceImpl.java

```diff
@@ -70,7 +74,8 @@ public void removeReviewById(ReviewToRemoveDto reviewRemoveDto) {
     public List<ReviewAdminResponse> getReviewsForProductAllInf(String productSkuCode) {
         List<Review> reviews = repository.findReviewsByProductSkuCode(productSkuCode);
         if (reviews.isEmpty()) {
-            throw new DataNotFoundException("Review for product with sku code = " + productSkuCode + " isn't exist.");
+            log.info("[ON getReviewsForProductAllInf]:: Review for product with sku code = {} isn't exist.", productSkuCode);
+            return new ArrayList<>();
         }
         return reviews.stream().map(review -> mapper.map(review, ReviewAdminResponse.class)).toList();
```

Comment:
the same, we can simple add filter method to stream to avoid NullPointerException

Reply:
done